<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>SharpenWidget</class>
 <widget class="QWidget" name="SharpenWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>66</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>3</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QSlider" name="sldStrength" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>10</number>
       </property>
       <property name="pageStep" >
        <number>1</number>
       </property>
       <property name="value" >
        <number>2</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance" >
        <bool>false</bool>
       </property>
       <property name="invertedControls" >
        <bool>false</bool>
       </property>
       <property name="tickPosition" >
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval" >
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spnStrength" >
       <property name="maximum" >
        <number>10</number>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="value" >
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnOk" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <widget class="QCheckBox" name="chkPreview" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text" >
      <string>&amp;Preview</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Sharpen &amp;Strength</string>
     </property>
     <property name="buddy" >
      <cstring>spnStrength</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>sldStrength</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spnStrength</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>328</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel" >
     <x>388</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnStrength</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sldStrength</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>388</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel" >
     <x>328</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
